foo.rb

```diff
@@ -1,6 +1,5 @@
 class Block
   def foo
-    puts "one"
-    yield
```

Comment:
nope, you're doing it right. your expected output is missing the second `\n` character that `puts` appends